app/models/manageiq/providers/nuage/network_manager/event_catcher/runner.rb

```diff
@@ -17,6 +17,7 @@ def monitor_events
     event_monitor_handle.start do |event|
       event_monitor_running
       @queue.enq(event)
+      sleep_poll_normal
```

Comment:
Didn't we test this long ago already and it didn't help much? I'm surprised why it would help now. What's the throughput now?

Reply:
There were so many things that I have removed for this PR that it really hurt. The most important one was the batched processing which I thought will help the most, but there are actually plenty of sync issues. In the end, this is what was needed. I know, my LOC metric will not be good this time 😄.

Reply:
@gberginc - How will a sleep address the delay?

Reply:
Problem is that without this delay, the processing of events does not start. With this sleep, we allow the other thread to wake up and do the processing. This is how I understood it.